.gitignore

```diff
@@ -3,6 +3,7 @@
 /dist/*.ico
 /dist/*.map
 /dist/*.svg
+/dist/*.html
```

Comment:
Just saw that `.go` file are committed into the folder. Just ignore my question 😄